swap/Cargo.toml

```diff
@@ -36,7 +36,7 @@ torut = { version = "0.1" }
 tracing = { version = "0.1", features = ["attributes"] }
 tracing-core = "0.1"
 tracing-futures = { version = "0.2", features = ["std-future", "futures-03"] }
-tracing-log = "0.1"
+tracing-log = { version = "0.1" }
```

Comment:
?

Reply:
I was messing around with tracing and forgot to revert to the simplest version of this. I'll change it.